packages/api/src/sdk-bundle-api/sdk-to-bundle/session-data.ts

```diff
@@ -94,3 +99,15 @@ export interface EarlyRequest {
   startTime: number;
   duration: number;
 }
+
+export interface WebSocketConnectionData {
+  id: number;
```

Comment:
This just increments for each subsequent connection? If so we could maybe call it sequenceNumber to be clear (or add a comment). I guess technically since webSocketData is an array it's not needed?

Reply:
True, the ID is essentially the index in the array here, but for recording ws data I was planning on making each payload a map of id to WebSocketConnectionEvent[]. This structure makes it a bit cleaner to handle multiple ws connections to the same URL as well as payloads were no events were recorded for a connection. We could call this field `index` but I think `id` requires less explanation and makes the data structures a bit more consistent.

I'll update the type to `SequenceNumber`.